target/linux/ath79/dts/qca9533_tplink_tl-wr802n.dtsi

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
```

Comment:
Since the LED should be used for WiFi, drop led-running. By that, it won't stay on after successful boot, which might be mistaken for enabled wifi ...

Reply:
> 
> 
> > While testing I found out that at least for V2 the GPIO configuration for reset is different than what was found in ar71xx - GPIO12 active low. So I wonder if this isn't also the case for v1 - I would really appreciate a test for this one case. Before that discovery I thought that they differ only by SoC revision (AL1A vs BL1A) and LED colour (blue vs green).
> 
> That's why this should be added (rephrased) to the commit message. Then, I would have read it before writing the comments. ;-)
> 
> This should also be fixed for ar71xx in (a) separate commit(s).
> 
> Based on the differences and uncertainties, I would only merge v2 unless we can find a tester for v1.
> 
> > I have a backport for 19.07 ready and tested as well, pushed on tl-wr802n-ath79_19.07 branch on my Github.
> 
> I have not yet made up my mind whether I will backport this. In any case, I'd do the backport myself anyway, as this is effectively less work that reviewing somebody else's backport :-)
> I'd push this to separate staging for testing then, but I don't guarantee anything.

Sure, will do. Fixes for ar71xx are coming too (separately), at least for consistency.

Regarding backport - full understanding, given that it took me about 5 minutes to do this myself, having done support on master. Which was a kind of a 'porting speedrun' clocking in at 03:09 start to finish ;)

Reply:
Also see my comment on the trigger in 01_leds for this one :)

Reply:
Well, if the LED is really just a system LED, it would be better to have led-running. Having it off would look strange (like something is broken).

However, if somebody customizes the behavior, the led-running will be a annoying.

I'll leave it to you, so I don't have to decide :-)

Reply:
Actually I haven't had much problems if led-running was present and an override was made in /etc/config/system. The LED just goes on to steady state for a while before new trigger is set by UCI. So let's keep it, as it is most consistent with other devices supported by OpenWrt.